### ISSUE TITLE
Lost in translation (call of GetComputerNameW) 

### ISSUE THREAD

**Issue by vincss** [node-ffi/node-ffi#296]
I'm trying to call the function GetComputerNameW from kernel32.dll .

BOOL WINAPI GetComputerName(
  _Out_   LPTSTR  lpBuffer,
  _Inout_ LPDWORD lpnSize
);

I'm a bit lost, it seems that there is several way to pass object from C lib.
I've seen the use of : 
- new Buffer()
- Custom object named lptstr
- ...
Also on some examples, I've seen the use of :
ffi.Function
While in doc I only see ffi.Library

In short, it seems the code has evolved at lot since the writing of the tutorial, could it be updated ?
